<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraProperties</class>
 <widget class="QDialog" name="CameraProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>191</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Camera properties</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>441</width>
     <height>122</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="m_layout">
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item row="1" column="0">
     <widget class="QLabel" name="m_labelFOV">
      <property name="text">
       <string>FOV</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSlider" name="m_sliderFOV">
      <property name="minimum">
       <number>20</number>
      </property>
      <property name="maximum">
       <number>210</number>
      </property>
      <property name="singleStep">
       <number>10</number>
      </property>
      <property name="pageStep">
       <number>10</number>
      </property>
      <property name="sliderPosition">
       <number>60</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBelow</enum>
      </property>
      <property name="tickInterval">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="m_labelValue">
      <property name="text">
       <string>60</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="m_labelDegrees">
      <property name="text">
       <string>degrees</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QCheckBox" name="m_checkOrtho">
      <property name="text">
       <string>Orthographic</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
